docs/CONTRIBUTING.md

```diff
@@ -1,20 +1,24 @@
-Contributing to Forge
+Contributing to NeoForge
 =====================
 
-1) Keep patches to Minecraft classes together. If you need a lot of things done, you may either add to relevant forge classes or make a new class. Try not to spread out your patches across multiple disjoint lines, as this makes maintenance of your patches difficult.
+1) Keep patches to Minecraft classes together. If you need a lot of things done, you may either add to relevant NeoForge classes or make a new class. Try not to spread out your patches across multiple disjoint lines, as this makes maintenance of your patches difficult.
 
 2) TODO: Test Mods
 
-3) Follow the code style of the class you're working in (braces on newlines & spaces instead of tabs in Forge classes, inline brackets in patches, etc).
+3) Follow the code style of the class you're working in (braces on newlines & spaces instead of tabs in NeoForge classes, inline brackets in patches, etc).
+
+4) Sign the [CLA](https://cla-assistant.io/neoforged/NeoForge)
 
 For more information, refer to [the wiki][Wiki].
 
 
 Contributor License Agreement
 =============================
-- You grant Forge a license to use your code contributed to the primary codebase (everything **not** under patches) in Forge, under the LGPLv2.1 license.
-- You assign copyright ownership of your contributions to the patches codebase (everything under patches) to Forge, where it will be licensed under the LGPLv2.1 license.
+- You grant NeoForged a license to use your code contributed to the primary codebase (everything **not** under patches) in NeoForge, under the LGPLv2.1 license.
+- You assign copyright ownership of your contributions to the patches codebase (everything under patches) to NeoForged, where it will be licensed under the LGPLv2.1 license.
 
-"Forge" is Forge Development LLC, a legally incorporated entity in Oregon, USA.
+<br>
+"NeoForge" is NeoForged Development LLC, a legally incorporated entity.
```

Comment:
We are not, and claiming to be so is a federal crime (at least, in the USA), so this should be removed until we are.

Reply:
This line has been removed.